src/admin/models/user.ts

```diff
@@ -0,0 +1,25 @@
+export interface User {
```

Comment:
Sounds good to me. Can you create a card to capture that task? Just the card, we can organise pending work tomorrow